Dockerfile

```diff
@@ -17,14 +19,14 @@ ENV PATH="/virtualenv/bin:$PATH"
 RUN pip install \
     /srcdir \
     cx-Oracle \
-    ibm-db-sa \
     mysqlclient \
     psycopg2 \
     pymssql \
-    pyodbc
+    pyodbc \
+    $(arch | sed -e 's/x86_64/ibm-db-sa/g' -e 's/aarch64//g')
 
 RUN curl \
-    https://download.oracle.com/otn_software/linux/instantclient/instantclient-basiclite-linuxx64.zip \
+    https://download.oracle.com/otn_software/linux/instantclient/instantclient-basiclite-linux$(arch | sed -e 's/x86_64/x64/g' -e 's/aarch64/-arm64/g').zip \
```

Comment:
this could be a single expression: `sed -e 's/x86_64/x64/g; s/aarch64/-arm64/g'`

Reply:
You're right. Changed to single expression and commited.